src/main/java/com/recycle/buddy/model/Image.java

```diff
@@ -0,0 +1,23 @@
+package com.recycle.buddy.model;
+
+public class Image {
+
+    private String id;
+    private String base64;
```

Comment:
Could these fields be made final? Would there ever be a case where we would need to change their value? I.e can this class be made immutable?

Reply:
As I use Jackson and Spring boot , I have to keep this model and use setters, so fields can not be final.